src/events.js

```diff
@@ -297,6 +297,47 @@ const eventMap = {
     EventType: 'TransitionEvent',
     defaultInit: {bubbles: true, cancelable: true},
   },
+  // pointer events
+  pointerover: {
+    EventType: 'PointerEvent',
+    defaultInit: {bubbles: true, cancelable: true},
+  },
+  pointerenter: {
+    EventType: 'PointerEvent',
+    defaultInit: {bubbles: false, cancelable: false},
+  },
+  pointerdown: {
```

Comment:
Not familiar why the decision was made to name it `keyDown` instead of `keydown` and if it applies here as well.

Reply:
I think we wanted to camelCase it similar to how react does 👍 I believe it applies here as well.

Reply:
Alright will do.

Reply:
Done